homeassistant/loader.py

```diff
@@ -599,9 +604,18 @@ def iot_class(self) -> str | None:
     @property
     def integration_type(
         self,
-    ) -> Literal["entity", "integration", "hardware", "helper", "system"]:
+    ) -> Literal[
+        "entity",
+        "device",
+        "hardware",
+        "integration",
```

Comment:
This one can't come back anymore can it, since hub is the default? 
```suggestion
```

Reply:
> This one can't come back anymore can it, since hub is the default?

It never really existed, `integration` was a default, but wasn't allowed to be set in Hassfest (not part of the schema validation).

Therefore, the only occurrences of that `integration_type` were in our code only.

Reply:
Oh... good morning! Now I get the comment, it can be removed 👍 

Well spotted, thanks 👍